tests/test_util.cuh

```diff
@@ -121,6 +150,57 @@ inline uint64_t Murmur3HashHost(const uint64_t& key) {
   return k;
 }
 
+template <class K, class M, class V, size_t DIM = 16>
+void create_continuous_keys(K* h_keys, M* h_metas, V* h_vectors, int KEY_NUM,
+                            K start = 1) {
+  for (K i = 0; i < KEY_NUM; i++) {
+    h_keys[i] = start + static_cast<K>(i);
+    h_metas[i] = h_keys[i];
+    if (h_vectors != nullptr) {
+      for (size_t j = 0; j < DIM; j++) {
+        h_vectors[i * DIM + j] = static_cast<float>(h_keys[i] * 0.00001);
+      }
+    }
+  }
+}
+
+template <class K, class M, class V, size_t DIM = 16>
+void create_keys_in_one_buckets(K* h_keys, M* h_metas, V* h_vectors,
```

Comment:
Should remove the same func from `merlin_hashtable_test.cc.cu` and please notice to process the `DIM` properly.